### ISSUE TITLE
Add TravisCI configuration

### ISSUE THREAD

**Issue by mschwager** [duo-labs/dlint#10]
Once TravisCI is enabled for the repository we should add its configuration. Something like:

```
dist: xenial
language: python
python:
    - 2.7
    - 3.5
    - 3.6
    - 3.7
install:
    - pip install -r requirements.txt
    - pip install -r requirements-dev.txt
    - pip install -e .
script:
    - flake8
    - pytest --cov
```

**Comment by jordan-wright** [comment]
TravisCI is set up. And this seems like a reasonable CI config to me!

**Comment by mschwager** [comment]
Fixed by 7fd899e.

Thanks @jordan-wright!
